### PR TITLE
Add configs for php-cs-fixer and run

### PR DESCRIPTION
**Pull request by JellyBellyDev**
[v2/php-cs-fixer 9e508c9] Add configs for php-cs-fixer and run

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,5 @@
 /bin/php-cs-fixer
 /bin/phpspec
 /vendor
+.php_cs.cache
```
Comment (AlessandroMinoccheri) [inline .gitignore:5]: Is it necessary or we can add this into .gitignore?

Comment (garak) [reply .gitignore:5]: I would use `.*.cache` instead, to catch phpunit cache too (and other possible kinds of cache)

Comment (garak) [inline .gitignore:5]: I would use `.*.cache` instead, to catch phpunit cache too

Comment (JellyBellyDev) [reply .gitignore:5]: In this library there aren't phpunit! :P

**Pull request merged**